Repositories/UserRepository.cs

```diff
@@ -36,8 +37,9 @@ public async Task<User> GetById(string id)
         {
             try
             {
-                var userList = await GetCollection().AsQueryable().ToListAsync();
-                return userList.FirstOrDefault(x => x.Id == id.ToString() && x.Deleted == false);
+                var user = await GetCollection().AsQueryable()
+                    .FirstOrDefaultAsync(u => u.Id == id && !u.Deleted);
```

Comment:
@christopherplummer These should both be executed on the database server. Need to profile whether they need a compound index or whether having individual indexes on these properties is fine...